api/org/commcare/api/persistence/SqlHelper.java

```diff
@@ -219,7 +219,19 @@ public static void updateId(Connection c, String storageKey, Persistable p) {
      * @param id        sql record to update
      */
     public static void updateToTable(Connection connection, String tableName, Persistable persistable, int id) {
-        String query = "UPDATE " + tableName + " SET " + DatabaseHelper.DATA_COL + " = ? " + " WHERE " + DatabaseHelper.ID_COL + " = ?;";
+        String queryStart = "UPDATE " + tableName + " SET " + DatabaseHelper.DATA_COL + " = ? ";
+        String queryEnd = " WHERE " + DatabaseHelper.ID_COL + " = ?;";
+
+        HashMap<String, Object> map = DatabaseHelper.getMetaFieldsAndValues(persistable);
+        String[] fieldNames = map.keySet().toArray(new String[map.keySet().size()]);
+        Object[] values = map.values().toArray(new Object[map.values().size()]);
+
+        StringBuilder stringBuilder = new StringBuilder(queryStart);
+        for(String fieldName: fieldNames){
+            stringBuilder.append( ", " + fieldName + " = ?");
```

Comment:
Lint warning is suggesting that 
`.append( ", " + fieldName + " = ?");` 
be changed to 
`.append( ", ").append(fieldName).append(" = ?");`